lib/package.rb

```diff
@@ -54,9 +54,7 @@ def build_zip
   def load_or_create_manifest_node
     return load_manifest if @translation.resource.manifest.present?
 
-    manifest = Nokogiri::XML::Node.new('manifest', @document)
-    @document.root = manifest
-    manifest
+    @document.root = @document.create_element('manifest', 'xmlns' => 'https://mobile-content-api.cru.org/xmlns/manifest')
```

Comment:
is it possible to define `https://mobile-content-api.cru.org/xmlns/manifest` as a string constant somewhere? where should global constants go and what should be the syntax?

Reply:
That's a good question.  If we can't find the Rails norm you could just create a constants file within `lib/`

Reply:
I added it as a constant within XmlUtils for now